package.json

```diff
@@ -16,7 +16,12 @@
     "test": "mocha --reporter spec"
   },
   "dependencies": {
-    "yeoman-generator": "^0.17.0"
+    "ejs": "^2.3.0",
+    "html-wiring": "^1.0.0",
+    "mkdirp": "^0.5.0",
+    "to-pascal-case": "0.0.2",
```

Comment:
Maybe use https://github.com/blakeembrey/pascal-case, this one seems unmaintained and I don't think he's planning to release a 1.0 anytime soon. Also, use a caret ;)